integreat_cms/cms/linklists.py

```diff
@@ -38,22 +57,52 @@ class PageTranslationLinklist(Linklist):
     @classmethod
```

Comment:
We can remove `html_fields = ["content"]` now since it's inherited from the base class.